.clang-tidy

```diff
@@ -0,0 +1,38 @@
+---
+Checks:          'clang-diagnostic-*,clang-analyzer-*,-*,performance-faster-string-find,performance-for-range-copy,'
```

Comment:
Please change to yaml multplie line string https://yaml-multiline.info/. In that way we easily track changes
```
Checks: >
    -*,
    clang-analyzer-*,
    clang-diagnostic-*,
    performance-faster-string-find,
    performance-for-range-copy,
```

Reply:
```yaml
Checks:          'clang-diagnostic-*,clang-analyzer-*,-*,performance-faster-string-find,performance-for-range-copy,'
```

The configuration above is generated by `clang-tidy -header-filter='/(?!external)/.*' -checks='-*,performance-faster-string-find,performance-for-range-copy,' -warnings-as-errors='*' -dump-config`


```yaml
Checks: >
    -*,
    clang-analyzer-*,
    clang-diagnostic-*,
    performance-faster-string-find,
    performance-for-range-copy,
```

I just found that the above two are not the same. The order matters. `-*` will disable options ahead of it.

* In the first config, `clang-diagnostic-*,clang-analyzer-*` is disabled.

* The second config, placing `-*` at the front, will not disable `clang-diagnostic-*,clang-analyzer-*`.